### PR TITLE
Feat: Add test per million

### LINKED ISSUE #1032: Add test per million
**Is your feature request related to a problem? Please describe.**
Test per million is a clear indicator of how effective testing happens at the ground level.

**Describe the solution you'd like**
Add test per million header(preferably PPM)
**Describe alternatives you've considered**
Data is not available to do PPM at this point.
**Additional context**
image

### PR DESCRIPTION
**Pull request by kp666**
**Description of PR**

**Type of PR**

- [ ] Bugfix
- [* ] New feature

**Relevant Issues**  
Fixes #1032 .

**Checklist**

- [* ] Compiles and passes lint tests
- [ ] Properly formatted
- [* ] Tested on desktop
- [ ] Tested on phone

**Screenshots**
image

### CODE REVIEW

File: src/components/home.js [hunk 1]
```diff
@@ -27,7 +27,7 @@ function Home(props) {
   const [timeseriesMode, setTimeseriesMode] = useState(true);
   const [timeseriesLogMode, setTimeseriesLogMode] = useState(false);
   const [regionHighlighted, setRegionHighlighted] = useState(undefined);
-
+  const million = 1000000;
```
Comment (tannakartikey) [inline src/components/home.js]: can you convert it to `const MILLION = 1000000`?

File: src/components/data/state_wise_population_2011.json [hunk 2]
```diff
@@ -0,0 +1,40 @@
+{
```
Comment (tannakartikey) [inline src/components/data/state_wise_population_2011.json:1]: Can you comment in the file with the source of this data?

**Comment by tannakartikey** [comment]
@kp666 this PR is related to which issue can you please mention?

**Comment by kp666** [comment]
@tannakartikey  I haven't filed an issue

**Comment by kp666** [comment]
Added Issue #1032

**Review by tannakartikey — CHANGES_REQUESTED** [review]
@kp666 we will accept and review the PR in detail if the issue is accepted.
Follow the contribution guide for more details:
https://github.com/covid19india/covid19india-react/blob/master/CONTRIBUTING.md

Until then I have reviewed the PR at a glance.

File: src/components/data/state_wise_population_2011.json (hunk 2 above)
Comment (kp666) [reply src/components/data/state_wise_population_2011.json:1]: Could you propose how to comment on a json file?

Comment (kp666) [reply src/components/data/state_wise_population_2011.json:1]: The source of the data is district wise 2011 census data available here: https://www.kaggle.com/danofer/india-census
except andhra pradesh and telegana.
For Andhra Pradesh and Telegana I used data available on wikipedia.

**Review by jeremyphilemon — CHANGES_REQUESTED** [review]
Hey @kp666, this looks good! However, we don't want to show this data upfront on the table. We'd like to show it on the state page like this right below the `meta-secondary` component. Have it labelled as "Tests per million" rather than what I've put there: 

Screen Shot 2020-04-16 at 8 59 58 PM

Let me know if you can make those changes :)

**Comment by kp666** [comment]
@jeremyphilemon The intention from my side is not just to show data, but pressure states into rigorous testing. I feel the right place for that is upfront on the table, rather than hide it inside state page.

**Comment by jeremyphilemon** [comment]
I understand your sentiment, but it's not logistically possible to show all data in one place, which in this case is the table.

Furthermore, it is also counterintuitive to your cause because at the end of the day you're still just showing data, I fail to see how placing this in the table would "pressurize" the authorities into doing things than if it were placed elsewhere. 

Let me know what you think.

**Comment by kp666** [comment]
It is easier to compare states that do rigorous testing when data is listed on a list, rather than on the state page.  A sorted list like the one attached makes more sense to viewers than stats in a state page. 
Also Its always possible to change layout when you want to show more data. 

image

**Comment by jeremyphilemon** [comment]
Hey thanks for the PR! You can view "tests per million" on the individual state pages. Feel free to open PRs against available issues :)

**Pull request closed without merging**